### DISCUSSION TITLE
Decide what to use to set styles

### DISCUSSION THREAD

**Discussion by ykdojo** [ykdojo/OpenStream#42] [General]
Plain CSS? Tailwind? Something else?

**Comment by mehta-avi** [comment]
Personally, I would recommend using Tailwind since it's the most convenient and productive -- it would help us rapidly build out the UI while keeping styling consistent throughout. Using pure CSS, on the other hand, would require effort to maintain and organize as we scale, given that we have complete control of the styling.

**Comment by usmanfaki** [comment]
I think using Tailwind will be better than using plain CSS because of the reasons given by @mehta-avi. It will be a lot easier to read and maintain Tailwind than everyone writing their own class names in CSS.

**Comment by ykdojo** [comment]
Does it work well with React and Plasmo?

**Comment by mehta-avi** [comment]
Yes, Tailwind works pretty well with React -- I use it all the time as my frontend framework. I've never had the opportunity to work with Plasmo, but this is what the official documentation says: "Using TailwindCSS is super easy with Plasmo thanks to the built-in integration with PostCSS"

**Comment by ykdojo** [comment]
Seems like Tailwind is the right solution here then. I'll wait for a bit more to see if anyone else has something to add though.

**Comment by Devanshukoli** [comment]
Yup, I just read through the discussion. I also think it's better if we use **TailwindCSS** for it. :)

**Comment by ykdojo** [comment]
Okay let's go with Tailwind then!

**Comment by usmanbinnaeem** [comment]
seems like tailwindCSS will be great for fast UI development, offering flexibility and ease of use. It simplifies styling with utility classes, making it easier to maintain consistency across your extension. Ideal for projects where speed and design customization matter.
